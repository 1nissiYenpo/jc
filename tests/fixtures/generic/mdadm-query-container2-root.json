{"device":"/dev/md0","version":"ddf","raid_level":"container","total_devices":2,"working_devices":2,"container_guid":"4C696E75:782D4D44:DEADBEEF:00000000:502D04BE:255CF1BB","seq":"0000000e","virtual_disks":2,"member_arrays":"/dev/md/vol0 /dev/md/vol1","device_table":[{"number":null,"major":254,"minor":0,"device":"/dev/vda","raid_device":null},{"number":null,"major":254,"minor":16,"device":"/dev/vdb","raid_device":null}],"member_arrays_list":["/dev/md/vol0","/dev/md/vol1"]}
