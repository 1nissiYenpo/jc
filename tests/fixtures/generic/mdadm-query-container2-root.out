/dev/md0:
           Version : ddf
        Raid Level : container
     Total Devices : 2

   Working Devices : 2

    Container GUID : 4C696E75:782D4D44:DEADBEEF:00000000:502D04BE:255CF1BB
                  (Linux-MD 08/16/22 16:33:34)
               Seq : 0000000e
     Virtual Disks : 2

     Member Arrays : /dev/md/vol0 /dev/md/vol1

    Number   Major   Minor   RaidDevice

       -     254        0        -        /dev/vda
       -     254       16        -        /dev/vdb
